model/__init__.py

```diff
@@ -573,6 +575,9 @@ def deploy_delaying_proxy(self):
     def kubernetes_cluster_config(self):
         return self.snd.kubernetes_cluster_config
 
+    def disable_webhook_for_pr(self):
```

Comment:
I think we should name this a bit more expressively; e.g. disable_github_pr_webhooks; or github_pull_request_webhooks;

although most methods are not documented (because most attribute are rather self-explanatory), in this case it might make sense to add a docstring.